src/bin/rav1e.rs

```diff
@@ -208,27 +211,40 @@ fn do_encode<T: Pixel, D: Decoder>(
     for frame in frame_info {
       progress.add_frame(frame);
       if verbose {
-        eprintln!("{} - {}", frame, progress);
+        info!("{} - {}", frame, progress);
       } else {
+        // Print a one-line progress indicator that overrides itself with every update
         eprint!("\r{}                    ", progress);
       };
     }
 
     output.flush().unwrap();
   }
-  eprint!("\n{}\n", progress.print_summary());
+  if !verbose {
+    // Clear out the temporary progress indicator
+    eprint!("\r");
+  }
+  progress.print_summary();
   Ok(())
 }
 
 fn main() {
   better_panic::install();
+  init_logger();
 
   match run() {
     Ok(()) => {}
     Err(e) => error::print_error(&e),
   }
 }
 
+fn init_logger() {
+  if env::var("RUST_LOG").is_err() {
```

Comment:
@YaLTeR pretty_env_logger doesn't provide a facility for setting a default log level internally--it's intended to work with the env variable.